File: Java/src/RajyaSabha.java
 /*
Name:Shreyash Parsekar
Roll No:44
Batch-3
MCA-Part-1
*/
import java.util.ArrayList;
import java.util.List;

public class RajyaSabha {
    public List<MemberOfParliament> members = new ArrayList<>();

    public void addMember(MemberOfParliament member) {
        members.add(member);
    }

    public List<MemberOfParliament> getMembers() {
        return members;
    }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder("Rajya Sabha Members:\n");
        for (MemberOfParliament member : members) {
            sb.append(member.toString()).append("\n");
        }
        return sb.toString();
    }
}

File: C++/RajyaSabha.cpp
#include "RajyaSabha.h"

void RajyaSabha::addMember(MemberOfParliament member) {
    members.push_back(member);
}

std::string RajyaSabha::toString() {
    std::string result = "Rajya Sabha Members:\n";
    for (const auto& member : members) {
        result += member.toString() + "\n";
    }
    return result;
}
